src/index.ts

```diff
@@ -1,81 +1,128 @@
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- * @param logLevel is the log level (info, warn, error).
- */
-// prettier-ignore
-function createMessage (
-  message: unknown,
-  event: string,
-  messageEvent: string,
-  logLevel: LogLevel
-): string | Message {
-  const data: Message = {
-    timestamp: new Date().toISOString(),
-    log_level: logLevel,
-    event: event,
-    body: message,
-    message: {
-      event: messageEvent
+  /**
+   * Returns the current logger level.
+   *
+   */
+  public getLevel () {
+    let level: LogLevel | null = null
+
+    for (const [name, number] of Object.entries(logger.levels)) {
+      if (logger.getLevel() === number) {
+        level = name as LogLevel
+        break
+      }
     }
+    return level
   }
 
-  return JSON.stringify(data)
-}
+  /**
+   * Generates an output log with severity level DEBUG.
+   *
+   * @param {String} message is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public debug (message: string, payload: Object): void {
+    logger.debug(this._buildPayload(LogLevel.DEBUG, message, payload))
+  }
 
-/**
- * Logs an info log.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc)
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- */
-// prettier-ignore
-function info (message: unknown, event: string, messageEvent: string): void {
-  console.info(createMessage(message, event, messageEvent, LogLevel.INFO))
-}
+  /**
+   * Generates an output log with severity level INFO.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public info (message: string, payload: Object): void {
+    logger.info(this._buildPayload(LogLevel.INFO, message, payload))
+  }
 
-/**
- * Logs a warn log.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc)
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- */
-// prettier-ignore
-function warn (message: unknown, event: string, messageEvent: string): void {
-  console.warn(createMessage(message, event, messageEvent, LogLevel.WARN))
-}
+  /**
+   * Generates an output log with severity level WARN.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
```

Comment:
[Suggestion]
The message field isn't really the _"log event(i.e. request_started, request_finished and etc)."_. It is the message, and might also be _the log event_.

```suggestion
   * @param {String} message is the core information of your log entry. By reading it anyone should understand what the log entry is about.
   * @param {object} payload is a JSON object with all structured fields to be added to the log.
   */
```